feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/aggregate_forwarding_viable_test.go

```diff
@@ -365,6 +365,21 @@ func (tc *testArgs) configureATE(t *testing.T) {
 		}
 	}
 
+	// Disable FEC for 100G-FR ports because Novus does not support it.
+	p100gbasefr := []string{}
+	for _, p := range tc.atePorts {
+		if p.PMD() == ondatra.PMD100GBASEFR {
+			p100gbasefr = append(p100gbasefr, p.ID())
+		}
+	}
+
+	if len(p100gbasefr) > 1 {
```

Comment:
Should this be `len(p100gbasefr) > 0`?

Reply:
yes, you are correct. My bad, miss from my end